federation-js/src/service/printCoreSchema.ts

```diff
@@ -211,15 +228,15 @@ function printFederationTypeDirectives(type: GraphQLObjectType): string {
   const restEntries = Object.entries(restKeys);
 
   return (
-    `\n  @owner(graph: "${ownerService}")` +
+    `\n  @join__owner(graph: ${ownerService.toUpperCase()})` +
```

Comment:
I think we may want a separate `Graph` type (in TypeScript) that would encapsulate metadata about a graph, including the graph name and endpoints. That means we could store the graph name uppercased in there and don't have to remember to do that when printing.